### ISSUE TITLE
[Bug] Reordering Operations in Collection Unexpected Behavior

### ISSUE THREAD

**Issue by joe-highnote** [apollographql/apollo-studio-community#156]
When reordering operations in a collection, the drag-and-drop functionality does not always work as expected. The operation being dropped may jump to the bottom of the list or may jump to somewhere else entirely in the list or may not move at all. It is very unpredictable.

**Comment by joe-highnote** [comment]
@jgzuke Thank you for the update on this! However, I am still experiencing the same behavior in a new org that I have set up and duplicated a shared collection into. As an example of the behavior I just experienced, dragging an operation down the list so that it scrolls and then dropping it demonstrates the same unpredictability that I originally described. Not sure if this is because it is a duplicate of the collection that presented this issue or something else entirely.
